devops/ci/destroy/Jenkinsfile

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env groovy
+
```

Comment:
Rename this file to:
 devops/ci/single_node_deploy/Jenkinsfile_Deploy

Also, add copyright headers.